COMETwebapp/ViewModels/Components/ParameterEditor/ParameterBaseRowViewModel.cs

```diff
@@ -112,6 +112,33 @@ private void InitializesProperties(bool isReadOnly)
             this.Option = this.ValueSet.ActualOption is not null ? this.ValueSet.ActualOption?.Name : string.Empty;
             this.State = this.ValueSet.ActualState is not null ? this.ValueSet.ActualState.Name : string.Empty;
             this.Switch = this.ValueSet.ValueSwitch;
+            this.CalculateParameterGroupData();
+        }
+
+        /// <summary>
+        /// Calculate and set the values of <see cref="ParameterGroup"/> parameters
+        /// </summary>
+        private void CalculateParameterGroupData()

```

Comment:
please rename to  CalculateParameterGroupPath` as this is what is happening and also the name of the return value

Reply:
done